src/strings/en-US/PowerFxResources.resw

```diff
@@ -6361,6 +6361,14 @@
     <value>Check the types of the expressions inside the interpolated string.</value>
     <comment>How to fix message for an error</comment>
   </data>
+  <data name="ErrorResource_ErrEmptyIsland_ShortMessage" xml:space="preserve">
+    <value>Empty expressions cannot appear inside an interpolated string.</value>
+    <comment>Error message.</comment>
```

Comment:
@CarlosFigueiraMSFT , for localization teams, do we need to provide some context on "interpolated"? I know it's used in a few other messages here, but maybe we should comment that it's translated like C# docs do?

Reply:
It never hurts to provide additional context :-)

Reply:
Added some context